database/dynamodb.go

```diff
@@ -98,9 +96,9 @@ func ProcessGetFeatureFlagByHashKey(attributeName string, attributeValue string)
 	db := CreateDynamoDB()
 
 	input := &dynamodb.GetItemInput{
-		TableName: aws.String(GetFeatureFlagTableName()),
+		TableName: aws.String(GetTableName(utils.FF_TABLE_NAME)),
```

Comment:
Why are we changing this? @shubham-y

Reply:
The reason is instead of having two different functions `GetFeatureFlagTableName()` & `GetUserFeatureFlagTableName()` we can just have a common `GetTableName()` and pass the env variable name in it

Reply:
We can do that then again my APIs need to be changed, will do them in sometime or can we do this once all the code is being merged?

@shubham-y